.github/workflows/deploy.yml

```diff
@@ -1,30 +1,24 @@
 name: Deploy
-
 concurrency:
-  group: ${{ github.event.repository.name }}-deploy
+  group: deploy
```

Comment:
Are groups presumably inherently local to the repo, so putting the repo in is a waste of time?

Reply:
I think so yes, that seems clear from the docs. If I'm wrong I guess we'll find out!